tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import logging
+import boto3
+import json
+
+from remote_command_executor import RemoteCommandExecutor, RemoteCommandExecutionError
+
+# timeout in seconds
+INSTALLER_TIMEOUT = 1800
```

Comment:
I'd rather name this "STARRM_INSTALLATION_TIMEOUT" as it would be more self-explicative.